javatests/google/registry/flows/FlowTestCase.java

```diff
@@ -308,6 +308,11 @@ public EppOutput runFlow() throws Exception {
     return runFlow(CommitMode.LIVE, UserPrivileges.NORMAL);
   }
 
+  /** Shortcut to call {@link #runFlow(CommitMode, UserPrivileges)} as super user and live run. */
+  public EppOutput runFlowSuperuser() throws Exception {
```

Comment:
If this is only used in one place, I'm not sure it's worth having a separate aliasing method just for this - calling runFlow(CommitMode.LIVE, UserPriviliges.SUPERUSER) is not much longer and just as clear.

If you want to keep it, I'd rename to runFlowAsSuperuser() so it's a bit more explicit about what the superuser suffix means.  And actually, it turns out DomainAllocateFlowTest already defines a runFlowAsSuperuser():
https://github.com/google/nomulus/blob/f0d2f96c26ac30686a436f2bcd8c661b596bad19/javatests/google/registry/flows/domain/DomainAllocateFlowTest.java#L216

So preferably, if keeping this around, we'd reconcile the two to just have one definition.